pkg/kv/kvserver/metrics.go

```diff
@@ -68,6 +68,12 @@ var (
 		Measurement: "Replicas",
 		Unit:        metric.Unit_COUNT,
 	}
+	metaUninitializedCount = metric.Metadata{
+		Name:        "replicas.uninitialized",
+		Help:        "Number of uninitialized replicas, this does not include uninitialized replicas that can lie dormant in a persistent state.",
```

Comment:
> does not include uninitialized replicas that can lie dormant in a persistent state

Interesting -- Mind explaining how? Are uninitialized replicas ever persisted?

Reply:
This is from: https://github.com/cockroachdb/cockroach/issues/73368. @nvanbenschoten mentions 

> uninitialized replicas can lie dormant in persistent state without any corresponding in-memory representation